cli/lib/kontena/cli/nodes/update_command.rb

```diff
@@ -5,14 +5,18 @@ class UpdateCommand < Kontena::Command
 
     parameter "NODE_ID", "Node id"
     option ["-l", "--label"], "LABEL", "Node label", multivalued: true
+    option "--availability", "AVAILABILITY", "Node scheduling availability"
 
     def execute
       require_api_url
       require_current_grid
       token = require_token
 
       data = {}
-      data[:labels] = label_list if label_list
+      node = client.get("nodes/#{current_grid}/#{node_id}")
+      # Send the previous labels if none given
+      data[:labels] = label_list.empty? ? Array(node['labels']) : label_list
```

Comment:
It can just omit the `labels` from the POST if empty, it doesn't need to roundtrip them.... I think?

Similar fix in https://github.com/kontena/kontena/pull/2504#discussion_r124803890, needs an issue, I guess